src/styles/index.css

```diff
@@ -50,6 +50,42 @@
   }
 }
 
+/*  ANES Gradient Background Colors */
+@utility bg-gradient-blue-purple {
```

Comment:
FYI: 
These gradient colors might be tweaked later on. I decided to cherry-pick the example pages that display each color into a new branch for Kat to review: https://github.com/SU-SWS/anes-nextjs/pull/12

That way the random gradient functionality can still be reviewed here in this PR 😬